tests/test_context.py

```diff
@@ -56,11 +55,50 @@ def test_async_wake_interval(device):
     """Test read-write property async_wake_interval."""
     with Context(device) as context:
         context.async_wake_interval = 42
-        assert isclose(context.async_wake_interval, 42)
+        assert context.async_wake_interval == 42
+
+
+# THIS DOES NOT WORK, PLEASE HELP!
+
+# def test_is_supported(device):
+#     with Context(device) as context:
+#         assert context == current_context()
+#         with context:
+#             MONO42 = context.is_supported("MONO", "42")
+
 
+def test_available_resamplers(device):
+    """Test available_resamplers"""
+    with Context(device) as context:
+        assert context == current_context()
+        assert len(context.available_resamplers) >= 0
+
+
+def test_default_resampler_index(device):
+    """Test default_resampler_index"""
+    with Context(device) as context:
+        assert context == current_context()
+        assert context.default_resampler_index >= 0
```

Comment:
@McSinyx also this

Reply:
I think the best we can do here is asserting for if `0 <= default_resampler_index < len(available_resamplers)`.

Reply:
Gentil ping on this too.

Reply:
really?

Reply:
What do you mean by really?

Reply:
idk, since I cherry-pick ur commit for enum handling, things havent updated. I have already assert it for if `0 <= default_resampler_index < len(available_resamplers)`

Reply:
Strange, just do it again then.